packages/itwin/tree-widget/README.md

```diff
@@ -178,49 +205,99 @@ function CustomModelsTreeComponent({ imodel, viewport, getSchemaContext, selecti
+      {...modelsTreeProps}
+      getSchemaContext={getSchemaContext}
+      selectionStorage={selectionStorage}
+      imodel={imodel}
+      treeRenderer={(props) => <VisibilityTreeRenderer {...props} {...rendererProps} />}
+    />
+  );
+}
+
+<CustomModelsTreeComponent
+  selectionStorage={unifiedSelectionStorage}
+  imodel={imodel.imodel}
+  viewport={testViewport}
+  getFilteredPaths={async ({ createInstanceKeyPaths }) => {
+    return createInstanceKeyPaths({
+      // list of instance keys representing nodes that should be displayed in the hierarchy
+      targetItems: [imodel.physicalModel],
+    });
+  }}
+/>,
 ```
 
+<!-- END EXTRACTION -->
+
 The `ModelsTree` component displays a message when too many matches are found while filtering the tree; for this reason, it is recommended to throw `FilterLimitExceededError` that is provided by this package when the displayed subset is too large. Typically, this error is thrown when there are more than 100 matches. The error is cleared when a new reference for `getFilteredPaths` is provided.
 
 When a filter is provided or instance focus mode is used, the hierarchy automatically expands to show the targeted nodes. This might not be desirable when displaying a subset of the hierarchy and can be disabled by adding the `autoExpand: false` option to each path returned by `getFilteredPaths`:
 
+<!-- [[include: [Presentation.TreeWidget.GetFilteredPathsComponentExample, Presentation.TreeWidget.GetFilteredPathsWithOptionsExample], tsx]] -->
+<!-- BEGIN EXTRACTION -->
+
 ```tsx
-const getFilteredPaths = async ({ createInstanceKeyPaths }) => {
-  const paths = await createInstanceKeyPaths({ keys: myInstanceKeys });
-  // disable auto-expansion
-  return paths.map((path) => ({ path, options: { autoExpand: false } }));
-};
+type useModelsTreeProps = Parameters<typeof useModelsTree>[0];
+
+interface CustomCategoriesTreeProps {
+  getFilteredPaths: useModelsTreeProps["getFilteredPaths"];
+  viewport: Viewport;
+  selectionStorage: SelectionStorage;
+  imodel: IModelConnection;
+}
+
+function CustomModelsTreeComponent({ getFilteredPaths, viewport, selectionStorage, imodel }: CustomCategoriesTreeProps) {
+  const { modelsTreeProps, rendererProps } = useModelsTree({ activeView: viewport, getFilteredPaths });
+
+  return (
+    <VisibilityTree
+      {...modelsTreeProps}
+      getSchemaContext={getSchemaContext}
+      selectionStorage={selectionStorage}
+      imodel={imodel}
+      treeRenderer={(props) => <VisibilityTreeRenderer {...props} {...rendererProps} />}
+    />
+  );
+}
+
+<CustomModelsTreeComponent
+  selectionStorage={unifiedSelectionStorage}
+  imodel={imodel.imodel}
+  viewport={testViewport}
+  getFilteredPaths={async ({ createInstanceKeyPaths }) => {
+    const paths = await createInstanceKeyPaths({ targetItems: [imodel.physicalModel] });
+    return  paths?.map((path) => ({ path: Array.isArray(path) ? path : path.path, options: { autoExpand: false } }));
+  }}
 ```
 
+<!-- END EXTRACTION -->
```

Comment:
I think all this section can be removed. With the recent changes, the `createInstanceKeyPaths` function that's provided to `getFilteredPaths`, doesn't add the `autoExpand` prop at all, so by default nothing will be auto-expanded.

Reply:
Removed the section